x/cellarfees/keeper/auction.go

```diff
@@ -5,16 +5,19 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	auctiontypes "github.com/peggyjv/sommelier/v4/x/auction/types"
+	"github.com/peggyjv/sommelier/v4/x/cellarfees/types"
 )
 
 // Attempts to start an auction for the provided denom
 func (k Keeper) beginAuction(ctx sdk.Context, denom string) (started bool) {
 	activeAuctions := k.auctionKeeper.GetActiveAuctions(ctx)
 
 	// Don't start an auction if the denom has an active one
-	for _, auction := range activeAuctions {
-		if denom == auction.StartingTokensForSale.Denom {
-			return false
+	if len(activeAuctions) > 0 {
+		for _, auction := range activeAuctions {
```

Comment:
Does this loop not exit immediately on a zero-length slice?

Reply:
IIRC there was some bug that only went away when I added this but the integration tests pass without it now so idk.